skxray/correlation.py

```diff
@@ -1,13 +1,16 @@
 # ######################################################################
-# Original code(in Yorick):                                            #
+# Copyright (c) 2014, Brookhaven Science Associates, Brookhaven        #
+# National Laboratory. All rights reserved.                            #
+#                                                                      #
+# multi_tau_auto_corr Original code(in Yorick):                        #
 # @author: Mark Sutton                                                 #
 #                                                                      #
+# two_time_corr Original code                                          #
+# @author: Yugang Zhang                                                #
```

Comment:
Do these attributions belong in the docstrings of `two_time_corr` and `multi_tau_auto_corr` instead of in the license header?